server/server.go

```diff
@@ -142,6 +193,14 @@ func (s *RpcEndPointServer) HandleHttpRequest(respw http.ResponseWriter, req *ht
 	request.process()
 }
 
+func (s *RpcEndPointServer) handleDrain(respw http.ResponseWriter, req *http.Request) {
```

Comment:
Who will call this? Pre stop hook?

I thought that we only need to handle SIGTERM and then start returning errors on  /health